crates/viewer/re_viewer/src/app_state.rs

```diff
@@ -466,41 +470,6 @@ impl AppState {
         // Process deferred layout operations and apply updates back to blueprint
         viewport.update_and_sync_tile_tree_to_blueprint(&ctx);
 
-        {
-            // We move the time at the very end of the frame,
-            // so we have one frame to see the first data before we move the time.
```

Comment:
I don't quite understand this, even though I suspect I wrote it.

The "first" data will only be visible if we move/update the current time so that the latest-at query includes the first data.

Sure, if we receive a few video frames during one app update, the new code will not show the first frame of the video, but I don't think that's wrong, nor do I think the old code would have shown the first frame of the video either.